testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/migration/MigrationTest.java

```diff
@@ -82,6 +83,7 @@ public void migration4_xTest() throws Exception {
 
         // Always test offline-token login during migration test
         testOfflineTokenLogin();
+        testExtremelyLongClientAttribute(migrationRealm);
```

Comment:
I wonder why the test is added just to `migration4_xTest` and `migration9_xTest`. Shouldn't we rather put inside `testMigrationTo12_x` method?

Reply:
The long value of client attribute is only available since Keycloak 3.4.1, after KEYCLOAK-4231 changes. I've thus only put the long value into two newer realm jsons (4.8.3, 9.0.3). `testMigrationTo12_x()` is called from the methods testing migration from 1,*, 2.*, and 3.* as well. Thus I wanted to keep it only for 4.* and 9.* and kept it local here

Reply:
Makes sense, we need to make sure not to forget put it also to every newer version.